### ISSUE TITLE
In-Game: [PS5] Memory leaks when closing videos and deleting custom surfaces

### ISSUE THREAD

**Issue by Dopply** [YoYoGames/GameMaker-Bugs#2387]
### Description
When a video is closed on the PS5 platform, a roughly 5MB increase in memory is observed. This increased never decreases, even between room changes. I do not believe this issue is limited strictly to videos - it seems the PS5 platform does not properly clear surfaces from texture memory. I get a similar leak with my pause menu in my game, which saves the application surface as a sprite, applies a blur, and draws it. The sprite is deleted and surfaces are properly cleared afterwards, just like in this example, but the texture memory is never freed.

This leak does not appear on other platforms, best I can tell.

### Steps To Reproduce
1. Run project on PS5 with debugger attached.
2. Press X button to create the video object which will play.
3. Either wait for the video to finish or rapidly skip the video with the touchpad.
4. Observe increase in memory which never decreases.

(Sample project is a .yymps in the Attached Files folder.)

[61a9d6de-5d43-4664-8acb-3f95d72cb435](https://api.gamemaker.io/api/github/downloads/61a9d6de-5d43-4664-8acb-3f95d72cb435)

**Comment by caitlinrmcintyre** [comment]
Verified fix in IDE v2024.200.0.456  Runtime v2024.200.0.468, thanks.

**Comment by FMVInteractive** [comment]
There still appears to be a PS5 specific memory leak in version 2024.13.1.242

We make FMV games so they are very heavy on the video playback. Our latest game, made in the LTS version, experienced a memory leak when playing videos on PS5. This was not present in the PS4, Xbox, or Windows versions of the game. We switched to version 2024.13.1.242 thinking this was fixed, but we're still experiencing the exact same memory leak. After about 370-376 videos play, we get a blank black screen and no audio when trying to play more videos.
